<?xml version="1.0"?>

<launch>
    <node name="imau_control_node" pkg="trajectory_ros_control"  type="imau_control_node" 		output="screen" required="true">
    <param name="contr_name" type="string" value="imau_elevator_controller" />
    </node>

 <!--   <node name="imau_control_node" pkg="trajectory_ros_control"  type="imau_control_node" 		output="screen" required="true">
    <param name="contr_name" type="string" value="imau_right_gripper_controller" />
    </node>

    <node name="imau_control_node" pkg="trajectory_ros_control"  type="imau_control_node" 		output="screen" required="true">
    <param name="contr_name" type="string" value="imau_left_gripper_controller" />
    </node>-->








     <!-- Load the default controllers -->
  <!-- <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
	  output="screen" args="joint_state_controller imau_elevator_controller" /> -->


</launch>
